all_collections/_courses/info-privacy-policies-and-enhancing-tech.md

```diff
@@ -0,0 +1,16 @@
+---
+ref: info-privacy-policies-and-enhancing-tech
```

Comment:
γιατί έχει ινφο μπροστά? το "και" το θέλουμε?

Reply:
Ο τίτλος του μαθήματος στην αγγλική του μορφή στο di.ionio.gr είναι Information Privacy Policies and Enhancing Technologies. Εγώ θα το άφηνα το "και" δεδομένου ότι υπήρχε και σε αρκετά ακόμα μαθήματα με σύνθετους τίτλους. Εσείς τι θέλετε να κάνω;

Reply:
Συμφωνώ και άλλωστε δεν είναι το [μόνο](https://github.com/ioniodi/sitegr/blob/master/all_collections/_courses/social-and-legal-issues-of-icts.md) που ακολουθεί αυτό το μοτίβο. Αν αποφασίσουμε να το αλλάξουμε, μπορούμε να το δούμε συνολικά για όλα τα μαθήματα και όχι στα πλαίσια του μεμονωμένου αυτού PR.